tests/foreman/api/test_rhc.py

```diff
@@ -0,0 +1,122 @@
+"""Test class for RH Cloud connector - rhc
+
+:Requirement: Remoteexecution
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: RHCloud-CloudConnector
+
+:Team: Platform
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+import pytest
+from fauxfactory import gen_string
+from wait_for import wait_for
+
+from robottelo import constants
+from robottelo.utils.issue_handlers import is_open
+
+
+@pytest.fixture(scope='module')
+def fixture_enable_rhc_repos(module_target_sat):
```

Comment:
Do we need the crc_env='prod' check in this as well like we have in for ui? If yes then we can make a common fixture and move this fixture under pytest_fixtures.component.rh_cloud.

Reply:
If you do move the fixture to pytest_fixtures.component.rh_cloud can we please rename the fixture to just be called `enable_rhc_repos`

Reply:
The UI test is a part of a different pipeline, so we don't want that check here for API yet. @jameerpathan111

Reply:
@jyejare @shweta83 Because we only want one of these tests either UI/API to run in that pipeline. I'd like to keep it UI one as that's what most of our customers would use. 
And why do we need only one test to be part of that pipeline? Cause we want to use the same satellite in downstream iqe pipeline. There's no need for us to run two tests in that pipeline with stage and prod crc. The API test would be part of a normal automation run.